xml/System.Windows.Forms/ToolStripControlHost+ToolStripHostedControlAccessibleObject.xml

```diff
@@ -34,10 +34,9 @@
         <Parameter Name="toolStripControlHost" Type="System.Windows.Forms.ToolStripControlHost" />
       </Parameters>
       <Docs>
-        <param name="toolStripHostedControl">To be added.</param>
-        <param name="toolStripControlHost">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="toolStripHostedControl">The ToolStrip control hosted in the ToolStripControlHost container.</param>
```

Comment:
I'm not sure about the wording of these two parameters. How would you describe them with natural language?

Reply:
toolStripHostedControl - the control hosted in the ToolStripControlHost container
ToolStripControlHost - Container which hosts a control on the ToolStrip

Reply:
Ok, thanks! So the suggestions would be:

```suggestion
        <param name="toolStripHostedControl">The control hosted in the <see cref="T:System.Windows.Forms.ToolStripControlHost" /> container.</param>
```